lib/safer_rails_console/rails_version.rb

```diff
@@ -40,6 +40,14 @@ def six_or_above?
         @is_six_or_above = SaferRailsConsole::RailsVersion::RAILS_VERSION >= ::Gem::Version.new('6.0.0') if @is_six_or_above.nil?
         @is_six_or_above
       end
+
+      def seven_zero?
+        @is_seven_zero ||= Gem::Requirement.new('~> 7.0.0').satisfied_by?(SaferRailsConsole::RailsVersion::RAILS_VERSION)
```

Comment:
I don't believe this will memoize `false` correctly. Could you copy the patterns for how `six_zero?` and `six_or_above?` do it?

Reply:
Ah, sure. I'm used to using `defined?` for that check.